hubblestack_nova_profiles/security/meltdown_spectre.yaml

```diff
@@ -10,3 +10,70 @@ grep:
               grep_args:
                 - '-i'
       description: 'This checks the /proc/cpuinfo file for the pcid flag.'
+
+grep:
+  blacklist:
```

Comment:
Can we change this to whitelist matching? Just search for a `1` as the pattern. `match_on_file_missing` can be eliminated (default behavior), or kept to be explicit.

Reply:
After testing and running into a corner again, i think this has to stay a blacklist otherwise it's hard to reach check consistency with other Linux distros who don't implement debugfs paths.
For example, in CoreOS the _pti_ cpu flag is all we have to go on for having it enabled - but at the same time it could be disabled with kernel boot command line option - leaving us to check _if it has been disabled_ rather than _if it is active_.